### ISSUE TITLE
Does not work if CUDA v10 is installed.

### ISSUE THREAD

**Issue by QuickLearner171998** [clovaai/deep-text-recognition-benchmark#25]
How can we make this code compatible with CUDA v10  ??

**Comment by YacobBY** [comment]
Hey, I ran into this issue as well. After some tinkering I found out that this repo works with Cuda V10, you just have to build Pytorch from scratch, this is really easy though. Over at the Pytorch site there's a nice generator which will give you the commands to build Pytorch with your own python version and OS: https://pytorch.org/get-started/locally/

Personally (Ubuntu 19.04, Python 3,7) I executed these two commands in my Python VENV:

pip3 install https://download.pytorch.org/whl/cu100/torch-1.1.0-cp37-cp37m-linux_x86_64.whl
pip3 install https://download.pytorch.org/whl/cu100/torchvision-0.3.0-cp37-cp37m-linux_x86_64.whl

After this it worked.
